### PR TITLE
FS-1729; Add biosConfig Set command

### PR DESCRIPTION
**Pull request by jakeschuurmans**

### CODE REVIEW

File: cmd/bios/bios.go [hunk 1]
```diff
@@ -27,12 +29,24 @@ func CreateBiosControlCondition(ctx context.Context, action rctypes.BiosControlA
...
-	serverID, err := biosFlags.ParseServerID()
+	serverID, err := uuid.Parse(biosFlags.serverID)
 	if err != nil {
 		return err
 	}
 
-	params := rctypes.NewBiosControlTaskParameters(serverID, action)
+	var biosURL *url.URL
+	if action == rctypes.SetConfig {
+		if biosFlags.biosConfigURL != "" {
```
Comment (coffeefreak101) [inline cmd/bios/bios.go:39]: I don't think this if/else is necessary. Cobra should be returning an error if the user fails to set the flag, and the `url.Parse()` will return an error if the user provides an invalid URL.

Comment (jakeschuurmans) [reply cmd/bios/bios.go:39]: I agree its not necessary. I originally didn't include this check. I do think its worth keeping in the SUPER RARE case someone messes with the code and does something incorrectly so this error will trigger.

Comment (jakeschuurmans) [reply cmd/bios/bios.go:39]: So now that ive said that, I am willing to go either direction. Whatever you lean with, ill go with.

File: cmd/bios/set.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	"log"
+
+	mctl "github.com/metal-toolbox/mctl/cmd"
+	rctypes "github.com/metal-toolbox/rivets/condition"
+	"github.com/spf13/cobra"
+)
+
+var setCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set BIOS settings to something else",
```
Comment (coffeefreak101) [inline cmd/bios/set.go:13]: I think we could come up with a nicer description than, "change to something else". Something like "Set BIOS settings from remote config file".

File: cmd/bios/bios.go (hunk 1 above)
Comment (coffeefreak101) [reply cmd/bios/bios.go:39]: I still feel like it's unnecessary. The concern you have about a code change causing an issue would be better handled by a unit test. So I think we should remove the check, and if we're concerned about a chance causing an issue, then we add a unit test to validate the code's behavior.

**Review by coffeefreak101 — APPROVED** [review]

**Pull request merged**